test/column/resizable.test.tsx

```diff
@@ -0,0 +1,137 @@
+import { fireEvent } from '@testing-library/react';
+import type { Column } from '../../src';
+import { setup, getHeaderCells, getGrid } from '../utils';
+
+const pointerId = 1;
+
+// https://github.com/jsdom/jsdom/issues/2527
+class PointerEvent extends Event {
+  pointerId: number;
+  pointerType: string;
+  clientX: number;
+  buttons: number;
+
+  constructor(
+    type: string,
+    { pointerId, pointerType, clientX, buttons, ...rest }: PointerEventInit
+  ) {
+    super(type, rest);
+    this.pointerId = pointerId;
+    this.pointerType = pointerType;
+    this.clientX = clientX;
+    this.buttons = buttons;
+  }
+}
+
+// @ts-expect-error
+globalThis.PointerEvent = PointerEvent;
```

Comment:
Tests fail without this change. Had to suppress type errors. I don't think it is worth fixing them

![image](https://user-images.githubusercontent.com/1452717/128015804-cdf42148-3dee-4ff6-b290-07f4547d88ca.png)